automation/active_user_input/english_working-groups-config.yml

```
@@ -16,36 +16,41 @@ end-matter:
 # WG-Common
 
 wg-common:
-  include-wg: False
+  include-wg: True
   wg-fullname: Common Metrics WG
   repo-link: https://github.com/chaoss/wg-common
-  repo-branch: master
+  repo-branch: main
   focus-areas-location: wg-common/focus-areas
   focus-areas:
     contributions:
+    - clones.md
+    - language-distribution.md
```

```suggestion
    - programming-language-distribution.md
```

The metric has been recently renamed to match the actual name of the metric.

---

Thanks for fixing this!